src/config/ConfigService.js

```diff
@@ -20,21 +19,33 @@ export default class ConfigService extends Service {
 		// Initialization of dynamic configuration
 		const headerLogoFull = document.getElementsByName('header-logo-full')[0]?.content;
 		const headerLogoMini = document.getElementsByName('header-logo-minimized')[0]?.content;
+		const headerLogoFullDark = document.getElementsByName('header-logo-full-dark')[0]?.content;
+		const headerLogoMiniDark = document.getElementsByName('header-logo-minimized-dark')[0]?.content;
 		const title = document.getElementsByName('title')[0]?.content;
 		const customCSS = document.getElementsByName('custom-css-file')[0]?.content;
 
 		let dynamicConfig = {};
-		let brandImage = {};
-		// Add custom header full logo
+		// determine if any logos have been dynamically configured and assign brandImage property as an empty object
+		if (((headerLogoFull != undefined) && (headerLogoFull != "")) || ((headerLogoFullDark != undefined) && (headerLogoFullDark != "")) || ((headerLogoMini != undefined) && (headerLogoMini != "")) || ((headerLogoMiniDark != undefined) && (headerLogoMiniDark != ""))) {
+			dynamicConfig.brandImage = {};
+		}
+		// If header's full light logo has been configured, add it to dynamic config object
 		if ((headerLogoFull != undefined) && (headerLogoFull != "")) {
-			brandImage["full"] = headerLogoFull;
-			dynamicConfig["brand_image"] = brandImage;
+			Object.assign(dynamicConfig.brandImage, {"light": {"full": headerLogoFull}});
+		}
+		// If header's full dark logo has been configured, extend/add it to dynamic config's brandImage property
+		if ((headerLogoFullDark != undefined) && (headerLogoFullDark != "")) {
+			dynamicConfig.brandImage = {...dynamicConfig.brandImage, "dark": {"full": headerLogoFullDark}};
```

Comment:
@petrKavulok Please stick with the object assigning either one way or the other, but dont mix that together:

```
Object.assign(dynamicConfig.brandImage, {"dark": {"full": headerLogoFullDark}});
```